File: Backend/TopSys.TopConWeb.Domain/Entities/Lead/LeadContato.cs
namespace TopSys.TopConWeb.Domain.Entities.Lead
{
    public class LeadContato
    {
        public int Usina { get; set; }
        public int NumeroLead { get; set; }
        public int AnoLead { get; set; }
        public int Sequencia { get; set; }
        public string Nome { get; set; }
        public virtual CadastroGeral Funcao { get; set; }
        public int? FuncaoCodigo { get; set; } = 0;
        public int Ddd { get; set; }
        public int Telefone { get; set; }
        public int DddCelular { get; set; }
        public int Celular { get; set; }
        public string Email { get; set; }
    }
}

File: Frontend/src/app/classes/lead/lead-contato.ts
import { CadastroGeral } from "../cadastro-geral/cadastro-geral";
import { Usina } from "../usina/usina";

export class LeadContato {
    usina: Usina;
    numeroLead: number = 0;
    anoLead: number = 0;
    sequencia: number = 0;
    nome: string = '';
    funcao: CadastroGeral;
    ddd: number = 0;
    telefone: number = 0;
    dddCelular: number = 0;
    celular: number = 0;
    email: string = '';
}